Gemfile

```diff
@@ -4,7 +4,7 @@ ruby "2.7.2"
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.4.4'
 # Use sqlite3 as the database for Active Record
-gem 'sqlite3', '~> 1.3.12', group: [:test]
+# gem 'sqlite3', '~> 1.3.12', group: [:test]
```

Comment:
why did you comment this out?

Reply:
I removed the SQLite gem since it is not needed in the application because there really is no need for two database gems. Another reason was for this error; "C:/Ruby27-x64/lib/ruby/gems/2.7.0/gems/sqlite3-1.3.13-x64-mingw32/lib/sqlite3.rb:6:in `require': cannot load such file -- sqlite3/sqlite3_native (LoadError)".

Reply:
alright, I'm curious if you just revert the Gemfile and Gemfile.lock, does it work? are you using Mac or some other OS?   If it doesn't wonder if that is the reason.

Reply:
I'm working on a Windows laptop but that should not matter. Rails comes with the SQLite gem pre-installed but since you guys have the PostgreSQL gem already inside of the Gemfile, there is no need for the SQLite gem anymore. When I commented it out, the code coverage went up from 79% to 81.29%.